src/app/repositories/repo-table/repo-table.component.html

```diff
@@ -0,0 +1,8 @@
+<ibm-table-container>
+    <ibm-table-header>
+        <h4 ibmTableHeaderTitle>Carbon Repositories</h4>
+        <p ibmTableHeaderDescription>A collection of public Carbon repositories.</p>
+    </ibm-table-header>
+    <ibm-table [model]="model" [showSelectionColumn]="false" [striped]="false">
```

Comment:
Looks like there is the build error: `ERROR in src/app/repositories/repo-table/repo-table.component.html(6,16): Property 'model' does not exist on type 'RepoTableComponent'.`, since there isn't a model in your `repo-table.component.ts`